vars/stepResult.groovy

```diff
@@ -23,6 +23,7 @@ def call(Map config= [:]) {
    * config['junit_files'] The names of any available junit files.
    */
 
+    // node block runs this on different build agent
```

Comment:
Did you try removing the `node { }` to see why this needs one?  Perhaps/probably, we can narrow down the scope of the need for it, or remove it altogether armed with our more advanced knowledge of Jenkins Pipelines now.

Reply:
I think you mentioned in IM that the `node { }` isn't needed, or am I mis-recalling?